students/chelsea_nayan/lesson06/test_mailroom4.py

```diff
@@ -0,0 +1,27 @@
+# Unit testing for mailroom4.py
+
+from mailroom4 import list_thankyou, update_thankyou, check_thankyou, text_send
+
+
+def test1():
+    donor = 'Man Mannington'
+    assert donor in list_thankyou()
+
+def test2():
+    assert update_thankyou() is str
+
+def test3():
+    expected = 'Thank you Paul for your generous donation of $130.00.'
+    assert check_thankyou('Paul', '130')
```

Comment:
what is this testing? you're not using `expected` and your assert would fail since `check_thankyou` returns None